README.rst

```diff
@@ -1,10 +1,161 @@
+
+    git clone https://github.com/quipucords/camayoc.git
+    cd camayoc
+
+2. Create and activate a Python 3.6 virtual environment::
+
+   /usr/bin/python3.6 -m venv ~/envs/camayoc
+   source ~/envs/camayoc/bin/activate
+
+3. Install using make targets::
+
+    # To install for development, install all dependencies
+    make install-dev
+    # If you only want to run the test suite, you can skip
+    # a few dependencies with the plain "install" target
+    make install
+
+Configuration
+^^^^^^^^^^^^^
+
+Camayoc requires a configuration file to be installed to
+``$XDG_CONFIG_HOME/camayoc/config.yaml``. On most systems this will be
+``~/.config/camayoc/config.yaml``.
+
+The config file tells Camayoc about the hosts you would like to run tests
+against and the credentials to use with these hosts. It also tells Camayoc
+where the quipucords_ server you want to test if you are running
+quipucords_ tests.
+
+Camayoc contains test suites for both quipucords/rho and quipucords/quipucords
+projects. All portions of the test suite expect the config file format.  Any
+changes to the test suite that require changes to the config file format should
+be made with this in mind.
+
+There is an example annotated config file in ``camayoc/example_config.yaml``
+
+Configuring py.test
+^^^^^^^^^^^^^^^^^^^
+
+If you are testing machines that use ``https`` but do not have good
+certificates, it can get difficult to read the test results due to warnings
+about insecure connections. If this is a problem for you, an example
+``pytest.ini`` file is provided in the root of the ``Camayoc`` repository that
+disables these warnings. Additional configuration is possible through the
+``pytest.ini`` file, these options are documented at
+https://docs.pytest.org/en/latest/reference.html#ini-options-ref
+
+For example, if you always want to run ``py.tests`` in verbose mode, this can
+be done in the ``pytest.ini`` file.
+
+Running The Test Suite
+^^^^^^^^^^^^^^^^^^^^^^
+
+While Camayoc  contains test for rho_ and quipucords_, and these share
+a config file format, testing both rho_ and quipucords_ simultaneously is
+not a supported use case.
+
+Camayoc uses ``py.test`` as its test driver. The pytest project had excellent
+documentation available at https://docs.pytest.org/en/latest/contents.html
+
+Running Tests For rho_
+^^^^^^^^^^^^^^^^^^^^^^
+
+To run all the tests for rho_, first have rho_ installed and available in
+your ``$PATH``. This can be done by either ``rpm`` install to the system or by
+``pip`` installing rho_ from source into your Camayoc virtual
+environment. Then invoke the main test suite from the root directory of
+Camayoc with::
+
+    make test-rho
+
+There are some additional tests for rho located in ``camayoc/tests/remote/rho``
+that scan all machines listed in the inventory with all available network
+credentials. To run these, you similarly invoke::
+
+    make test-rho-remote
+
+Running Tests For quipucords_ and qpc_
```

Comment:
Reading qpc here kind of confuses me with test-qpc which means testing everything under qcs directory. I think we could leave the section title as `Running Tests For quipucords_`